### ISSUE TITLE
Check required CKEditor version

### ISSUE THREAD

**Issue by mlewand** [cksource/ckeditor-plugin-a11ychecker#23]
We should check what version of CKEditor is required in order to run Accessibility Checker. This information should be included into README.
